docs/advanced_topics/api/v2/usage.rst

```diff
@@ -0,0 +1,543 @@
+                    "slug": "blog",
+                    "first_published_at": "2016-09-21T13:54:00Z"
+                },
+                "title": "About"
+            },
+            {
+                "id": 10,
+                "meta": {
+                    "type": "standard.StandardPage",
+                    "detail_url": "http://api.example.com/api/v2/pages/10/",
+                    "html_url": "http://www.example.com/about/",
+                    "slug": "about",
+                    "first_published_at": "2016-08-30T16:52:00Z"
+                },
+                "title": "About"
+            },
+
+            ...
+        ]
+    }
+
+The ``?descendant_of`` filter also takes the id of a page but includes all
+descendants (children of children) instead of just directly children.
+
+Search
+------
+
+Passing a query to the ``?search`` parameter will perform a full-text search on
+the results.
+
+The query is split into "terms" (by word boundary), then each term is normalised
+(lowercased and unaccented).
+
+For example: ``?search=James+Joyce``
+
+Search operator
+^^^^^^^^^^^^^^^
+
+The ``search_operator`` specifies how multiple terms in the query should be
+handled. There are two possible values:
+
+ - ``and`` - All terms in the search query (excluding stop words) must exist in
+   each result
+ - ``or`` - At least one term in the search query must exist in each result
+
+The ``or`` operator is generally better than ``and`` as it allows the user to be
+inexact with their query and the ranking algorithm will make sure that
+irrelevant results are not returned at the top of the page.
+
+The default search operator depends on whether the search engine being used by
+the site supports ranking. If it does (Elasticsearch), the operator will default
+to ``or``. Otherwise (database), it will default to ``and``.
+
+For the same reason, it's also recommended to use the ``and`` operator when
+using ``?search`` in conjunction with ``?order`` (as this disables ranking).
+
+For example: ``?search=James+Joyce&order=-first_published_at&search_operator=and``
+
+Fields
+------
+
+By default, only a subset of the available fields are returned in the response.
+The ``?fields`` parameter can be used to both add additional fields to the
+response and remove default fields that you know you won't need.
+
+Additional fields
+^^^^^^^^^^^^^^^^^
+
+Additional fields can be added to the response by setting ``?fields`` to a
+comma-separated list of field names you want to add.
+
+For example, ``?fields=body,feed_image`` will add the ``body`` and ``feed_image``
+fields to the response.
+
+This can also be used across relationships. For example,
+``?fields=body,feed_image(width,height)`` will nest the ``width`` and ``height``
+of the image in the response.
+
+All fields
+^^^^^^^^^^
+
+Setting ``?fields`` to and asterisk (``*``) will add all available fields to be
+added to the response. This is useful for discovering what fields have been
```

Comment:
and asterisk -> an asterisk

will add all available fields to be added to the response -> will add all available fields to the response